Hotkeys

MajinBui opened the pull request:
Added functions and hotkeys for play/pause and stop/reset.  Some placeholders are used until further instructions.

apigarage commented on keymaps/pomodoro-app.cson:
```diff
@@ -9,3 +9,6 @@
 # 'atom-workspace':
 #   'ctrl-alt-o': 'pomodoro-app:toggle'
+'atom-workspace':
+  'ctrl-alt-j': 'atom-pomodoro-app:toggle'
```
for consistency, how about naming it "toggleTimer"?

MajinBui commented:
Currently, I have cleaned up the code to using a enum timerStateEnum:
https://github.com/MajinBui/atom-pomodoro-app/blob/hotkeys/lib/index.coffee#L26-L29

I also added buttons to the view:
https://github.com/MajinBui/atom-pomodoro-app/blob/hotkeys/lib/pomodoro-app-view.coffee#L10-L27
![button-pomodoro](https://cloud.githubusercontent.com/assets/8998363/10838623/54bf0aaa-7e9f-11e5-8435-b0fefa51ec9d.gif)

I am now trying to add event listeners to the button but it seems as though I might have to move the timer functions into the view.

MajinBui commented:
Never mind, I was able to add the event listeners in for the buttons.  Now, all that is left is the actual timer itself.  Preview:
![working-buttons](https://cloud.githubusercontent.com/assets/8998363/10838831/8b402520-7ea2-11e5-938d-7105febe5e18.gif)
https://github.com/MajinBui/atom-pomodoro-app/blob/hotkeys/lib/index.coffee#L38-L39

MajinBui commented:
TImer is complete.
![timer completed](https://cloud.githubusercontent.com/assets/8998363/10866543/5c0844d2-8009-11e5-88d0-47e88b86695e.gif)

MajinBui commented:
Yep, code related to debugging is now removed.
